libcontainerd/queue_unix_test.go

```diff
@@ -0,0 +1,31 @@
+package libcontainerd
```

Comment:
Oops; build on Windows is failing;

```
06:03:49 # github.com/docker/docker/libcontainerd
06:03:49 libcontainerd\queue_unix_test.go:12: undefined: queue
```

This file also needs the same build tags as `queue_unix.go`;

```go
// +build linux solaris
```